internal/builders/golang/build.go

```diff
@@ -0,0 +1,202 @@
+package golang
```

Comment:
Nitpicking: The language is called `Go` and not `golang`.

Reply:
I saw a few lines below it is handled as `go`. Is `package go` a reserved keyword?

Reply:
yeah, but it complains if I create a package named `go` for some reason.. heh

Reply:
Fine for me, was just a nitpick anyway. And no one love nitpickers ;)